### PR TITLE
remove errorneous error logging and db inserts in group experiment assign call

### PR DESCRIPTION
**Pull request by danoswaltCL**
https://github.com/CarnegieLearningWeb/UpGrade/issues/2147

### CODE REVIEW

File: backend/packages/Upgrade/src/api/services/ExperimentAssignmentService.ts [hunk 1]
```diff
@@ -863,36 +863,11 @@ export class ExperimentAssignmentService {
...
-    // exclude experiments which are not previously assigned and throw error
+    // exclude experiments which are not previously assigned
     const experimentToExclude = experiments.filter((experiment) => {
       return groupExperimentAssignedIds.indexOf(experiment.id) === -1;
     });
 
-    const experimentToExcludeIds = experimentToExclude.map((experiment) => experiment.id);
-
-    // throw error user group not defined and add experiments which are excluded
-    experimentToExclude.forEach(({ id, name }) => {
```
Comment (VivekFitkariwala) [inline backend/packages/Upgrade/src/api/services/ExperimentAssignmentService.ts:874]: We should keep the logger and comment out the insertion in the database.

**Review by VivekFitkariwala — DISMISSED** [review]

File: backend/packages/Upgrade/src/api/services/ExperimentAssignmentService.ts (hunk 1 above)
Comment (danoswaltCL) [reply backend/packages/Upgrade/src/api/services/ExperimentAssignmentService.ts:874]: it's not just the insert that is a problem, when i was testing this yesterday, the log and insert both are called for no reason for most users, regardless of if they have an invalid group or not.

Comment (danoswaltCL) [reply backend/packages/Upgrade/src/api/services/ExperimentAssignmentService.ts:874]: https://github.com/CarnegieLearningWeb/UpGrade/issues/2147#issuecomment-2515876690 i can demonstrate this in my local in mtg

Comment (danoswaltCL) [reply backend/packages/Upgrade/src/api/services/ExperimentAssignmentService.ts:874]: https://github.com/CarnegieLearningWeb/UpGrade/issues/2149

**Review by bcb37 — DISMISSED** [review]

**Review by bcb37 — APPROVED** [review]

**Pull request merged**